src/util/htmlToJson.ts

```diff
@@ -200,9 +200,35 @@ const himalayaToBlock = (nodes: HimalayaNode[]): Block | Block[] => {
   return result
 }
 
+/** Gets all unique tag lists that is included in the text that we import. */
+const allTagsFinder = (nodes: Element[]): string[] => {
+  const tagLists = nodes.reduce((accum, current) => {
+    if (current?.children?.length) return [...accum, current.tagName, ...allTagsFinder(current.children as Element[])]
+    return current.tagName ? [...accum, current.tagName] : [...accum]
```

Comment:
You can write this as a single ternary statement with multiple branches:

```
return a ? a1
	: b ? b1
  : c ? c1
  : d
```

In general you want to avoid multiple return statements in a function, unless it is a small short-circuiting return statement at the beginning of the function. A single exit point is easier to reason about.